### PR TITLE
Release v0.18.0-M6

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

**Comment by rossabaker** [comment]
@ChristopherDavenport Wading through the immense changelog, I wasn't sure which logger changes were user-facing.  You might remember, or I can dig while I'm fresh.  This bears mentioning in the release log, though it's not a tragedy if it happens after the release.

File: website/src/hugo/content/changelog.md [hunk 1]
```diff
@@ -8,6 +8,30 @@ Maintenance branches are merged before each new release. This change log is
...
+* Tested on Java 9.
+* `Message.withContentType` now takes a `Content-Type` instead of an
+  ``Option[`Content-Type`]``.  `withContentTypeOption` takes an `Option`,
+  and `withoutContentType` clears it.
+* `QValue` has an `HttpCodec` instance
+* `AuthMiddleware` never falls through.  See
+  [#1530](https://github.com/http4s/http4s/pull/1530) for more.
+* `ContentCoding` is no longer a `Registry`, but has an `HttpCodec`
+  instance.
+* Render a banner on server startup.  Customize by calling
```
Comment (jmcardon) [inline website/src/hugo/content/changelog.md]: Probably the most important change in this milestone.

File: website/src/hugo/content/changelog.md [hunk 2]
```diff
@@ -8,6 +8,30 @@ Maintenance branches are merged before each new release. This change log is
...
+  `withBanner(List[String])` or `withoutBanner` on the
+  `ServerBuilder`.
+* Parameterize `isZippable` as a predicate of the `Response` in `GZip`
+  middleware.
+* Add constant for `application/vnd.api+json` MediaType.
+* Limit memory consumption in `GZip` middleware
+* Add `handleError`, `transform`, and `bimap` to `EntityDecoder`.
```
Comment (jmcardon) [inline website/src/hugo/content/changelog.md]: and `transformWith`

File: website/src/hugo/content/changelog.md [hunk 3]
```diff
@@ -8,6 +8,30 @@ Maintenance branches are merged before each new release. This change log is
...
+* Dependency upgrades:
+  * fs2-0.10.0-M9
+  * fs2-reactive-streams-0.2.6
+  * jawn-fs2-0.12.0-M4
+  * specs2-4.0.2
+
```
Comment (jmcardon) [inline website/src/hugo/content/changelog.md]: I think we are missing CSRF middleware.

Comment (rossabaker) [reply website/src/hugo/content/changelog.md]: That's mentioned in 0.16.6.  The changelog is meant to be read top-down-to-your-version to account for the things that debuted in earlier versions.

**Pull request merged**